transaction/transactor.go

```diff
@@ -192,60 +178,3 @@ func Welcome(delegate http.Handler) http.Handler {
 			delegate.ServeHTTP(w, r.WithContext(ctx))
 		})
 }
-
-// Capture (for lack of a better name) captures context values of interest
-// from the incoming request. Unlike Welcome, values captured here are
-// intended to be used only throughout the gokit server flow: (request decoding, business logic, response encoding)
-func Capture(logger kitlog.Logger) kithttp.RequestFunc {
-	var transactionInfoLogger = logging.Info(logger)
-	return func(ctx context.Context, r *http.Request) (nctx context.Context) {
-		var tid string
-
-		if tid = r.Header.Get(HeaderWPATID); tid == "" {
-			tid = GenTID()
-		}
-
-		nctx = context.WithValue(ctx, ContextKeyRequestTID, tid)
```

Comment:
@johnabass looks like this is the breaking change. Is is ok if we move it to the `Welcome` func in this file?